ios-showcase-template/deviceTrust/DeviceTrustViewController.swift

```diff
@@ -114,12 +119,12 @@ class DeviceTrustViewController: UIViewController, UITableViewDataSource, UITabl
         
         // set the text colouring
         if detection.passed {
-            textView.text = detection.result.capitalized
+            textView.text = CHECKS_RESULT[detection.name]![detection.passed]!
             textView.textColor = GREEN_COLOR
             imageView.image = UIImage(named: "ic_verified_user_white")?.withRenderingMode(UIImageRenderingMode.alwaysTemplate)
             imageView.tintColor = GREEN_COLOR
         } else {
-            textView.text = detection.result.capitalized
+            textView.text = CHECKS_RESULT[detection.name]![!detection.passed]!
```

Comment:
I think this will always show the text for `true` because we're negating it? This should probably also be `CHECKS_RESULT[detection.name]![detection.passed]!` right?